<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolarAlignWidget</class>
 <widget class="QWidget" name="PolarAlignWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>25</height>
   </rect>
  </property>
        <layout class="QGridLayout" name="PAStateLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="PAc1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Capture</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" alignment="Qt::AlignHCenter">
          <widget class="KLed" name="PAcapture1Led" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAso1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Solve</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAsolve1Led" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAs1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Slew</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAslew1Led" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>             
          </widget>
         </item>
         <item row="0" column="3" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAc2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Capture</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAcapture2Led" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>             
          </widget>
         </item>
         <item row="0" column="4" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAso2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Solve</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAsolve2Led" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>             
          </widget>
         </item>
         <item row="0" column="5" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAs2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Slew</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAslew2Led" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>             
          </widget>
         </item>
         <item row="0" column="6" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAc3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Capture</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAcapture3Led" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>             
          </widget>
         </item>
         <item row="0" column="7" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAso3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Solve</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAsolve3Led" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>             
          </widget>
         </item>
         <item row="0" column="8" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAsetup">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Setup</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAsetupLed" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>             
          </widget>
         </item>
         <item row="0" column="9" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PAadjust">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>             
           <property name="text">
            <string>Adjust</string>
           </property>
          </widget>
         </item>
         <item row="1" column="9" alignment="Qt::AlignHCenter">
           <widget class="KLed" name="PAadjustLed" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>             
          </widget>
         </item>
        </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
